tools/WinObjC.Tools/WinObjC.Tools.nuproj

```diff
@@ -14,19 +14,18 @@
     </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Configuration">
-    <PackageId>WinObjC.Tools</PackageId>
-    <Title>WinObjC.Tools</Title>
+    <PackageId>winobjc-tools</PackageId>
+    <Title>winobjc-tools</Title>
     <Authors>Microsoft</Authors>
     <Owners>Microsoft</Owners>
-    <Summary>WinObjC.Tools</Summary>
-    <Description>WinObjC.Tools</Description>
+    <Summary>Command Line tools and Visual Studio Extensions for the WinObjC Project.</Summary>
+    <Description>Command Line tools and Visual Studio Extensions for the WinObjC Project. This package includes vsimporter, the tool to convert an existing xcode objective c project to a visual studio solution, as well as the Visual Studio Extensions for Objective C syntax highlighting and Nuget package creation. See https://github.com/Microsoft/WinObjC for more details.</Description>
     <PackageReleaseNotes>
     </PackageReleaseNotes>
```

Comment:
its 2017 or later so I didn't want to be too specific. I'll add a line to description though.